docs/release/release_0_3_0.md

```diff
@@ -0,0 +1,141 @@
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
+
+## New Features
+- Hook up reader plugins (#937)
+- Support for magicgui (#981)
+- Writer plugins (#1104)
+
+
+## Improvements
+- Generalize keybindings (#791)
+- Points view data refactor (#951)
+- Add magic name guessing (#1008)
+- Refactor labels layer mouse bindings (#1010)
+- Reduce code duplication for `_on_scale_change` & `_on_translate_change` (#1015)
+- Style refactor (#1017)
+- Add ScaleTranslate transform (#1018)
+- Add docstrings for all the Qt classees (#1022)
+- Sorting and disabling of hook implementations (#1023)
+- Plugin exception storage and developer notification GUI (#1024)
+- Refactor points layer mouse bindings (#1033)
+- Add chains of transforms (#1042)
+- Shapes layer internal renaming (#1046)
+- Internal utils / keybindings / layer accessories renaming (#1047)
+- Shapes layer mouse bindings refactor (#1051)
+- Make console loading lazy (#1055)
+- Remove scikit-image dependency (#1061)
+- All args to add_image() accept sequences when channel_axis != None (#1092)
+- Added documentation for label painting (#1094)
+- Finish layer mouse bindings refactor (#1121)
+- Enable volume rendering interpolation control (#1127)
+- Change image icon (#1128)
+- Refactor Vectors colors (#1130)
+- Reduce code duplication in Points (#1140)
+- Add keybinding to toggle selected layer visibility (#1147)
+- Only block in gui_qt or quit QApp when necessary (#1148)
+- Plugin `_HookCaller` and registration updates (#1153)
+- Reduce code duplication in points tests (#1154)
+- Color cycles properties return array (#1163)
+- Allow last added point to be deleted with backspace keybinding (#1164)
+- Extract plugin code to napari-plugin-engine (#1169)
+
+
+## Bug Fixes
+- Refactor cleanup, prevent leaked widgets, add viewer.close method (#1014)
+- Allow StringEnum to accept instances of self as argument (#1027)
+- Close the canvas used to retrieve maximum texture size. (#1028)
+- Fix points color cycle refresh (#1034)
+- Fix styles for Qt < 5.12.  Fix styles in manifest for pip install (#1037)
+- Fix label styles in contrast limit popup  (#1039)
+- Fix pyramid clipping (#1053)
+- Fix resources/build_icons:build_resources_qrc (#1060)
+- Add error raise in `Viewer._add_layer_from_data` (#1063)
+- Fix empty points with properties (#1069)
+- Fix image format (#1076)
+- Lazy console fixes (#1081)
+- Fix function and signature to match with code (#1083)
+- Fix LayerList and ListModel type checking (#1088)
+- Fix changing translate order when changing data dims (#1102)
+- Fix add_points_with_properties example (#1126)
+- Use mode='constant' in numpy.pad usage (#1150)
+- Fix canvas none after layer deletion (#1158)
+
+
+## Breaking API Changes
+- Allow add_path() to accept any layer-specific kwarg and rename to open_path() (#1111)
```

Comment:
This title needs editing now that it's `open` and not `open_path`

Reply:
done